src/current/_includes/releases/v25.1/v25.1.0-alpha.3.md

```diff
@@ -0,0 +1,76 @@
+## v25.1.0-alpha.3
+
+Release Date: January 15, 2025
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v25-1-0-alpha-3-backward-incompatible-changes">Backward-incompatible changes</h3>
+
+- Several metrics are redundant and have been removed. Below is a list mapping from each removed metric to an existing, identical metric. [#138786][#138786]
+  - Removed `sql.schema_changer.running`, which is redundant with `jobs.schema_change.currently_running`.
+  - Removed `sql.schema_changer.successes`, which is redundant with `jobs.schema_change.resume_completed`.
+  - Removed `sql.schema_changer.retry_errors`, which is redundant with `jobs.schema_change.resume_retry_error`.
+  - Removed `sql.schema_changer.permanent_errors`, which is redundant with `jobs.schema_change.resume_failed`.
+
+<h3 id="v25-1-0-alpha-3-general-changes">General changes</h3>
+
+- When changefeeds are created with a resolved option lower than the `min_checkpoint_frequency` option, a notice is printed to inform the user since this may lead to unexpected behavior. [#138181][#138181]
+
+<h3 id="v25-1-0-alpha-3-sql-language-changes">SQL language changes</h3>
+
+- Users can always see and control (pause/resume/cancel) jobs which they own. [#138178][#138178]
+- Provide different options for `CREATE LOGICALLY REPLICATED TABLE`. [#138244][#138244]
+- `CHANGEFEED`s using named external connections now automatically update their configuration when the connection configuration changes. [#138237][#138237]
+- Added support for `DO` statements embedded within PL/pgSQL routines and other `DO` statements. `DO` statements execute a block of code inline as an anonymous function. Currently, only a PL/pgSQL body is allowed. [#138709][#138709]
+- Added support for `DO` statements in SQL, which allow a PL/pgSQL code block to be executed inline. [#138709][#138709]
+- `DO` statements can now be used as body statements in a SQL routine to execute PL/pgSQL code inline. [#138709][#138709]
+
+<h3 id="v25-1-0-alpha-3-operational-changes">Operational changes</h3>
+
+- If a row level TTL job is scheduled to run and the previous scheduled job for that table is still running, the scheduled run will now be skipped rather than waiting for the previous job to complete. [#138336][#138336]
```

Comment:
```suggestion
- If a row-level TTL job is scheduled to run and the previous scheduled job for that table is still running, the scheduled run will now be skipped rather than waiting for the previous job to complete. [#138336][#138336]
```